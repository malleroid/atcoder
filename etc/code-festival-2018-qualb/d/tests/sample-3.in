3 2 2
111111 1
999999 1
