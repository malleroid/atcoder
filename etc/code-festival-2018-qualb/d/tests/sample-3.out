666665.9999999997
